docs/install/windows_setup.md

```diff
@@ -114,81 +114,166 @@ For CPU-only package:
   install.packages("mxnet")
 ```
 
-For GPU-enabled package:
+#### Building MXNet from Source Code(CPU)
+1. Clone the MXNet github repo.
```

Comment:
View the markdown output. This numbering is wrapping instead of using new lines.